yarn.lock

```diff
@@ -1606,13 +1606,21 @@
     "@phosphor/signaling" "^1.2.3"
     "@phosphor/widgets" "^1.7.1"
 
-"@phosphor/disposable@^1.1.2", "@phosphor/disposable@^1.1.3":
+"@phosphor/disposable@^1.1.3":
```

Comment:
Note the duplication of disposable here (1.2.0 below).

Reply:
I have a commit that updates all the `@phosphor` libraries. I should push it because I think it resolves this duplication.